app/models/filters/hud_filter_base.rb

```diff
@@ -16,6 +16,16 @@ class HudFilterBase < FilterBase
     # Provide a mechanism to limit relevant project types
     attribute :relevant_project_types, Array, default: []
 
+    def initialize(attributes = {})
+      # Extract user from attributes hash if present
+      user = attributes[:user] if attributes.is_a?(Hash) || attributes.respond_to?(:[])
```

Comment:
I'd guess the following is sufficient:
```suggestion
      user = attributes[:user] if attributes.respond_to?(:[])
```